### PR TITLE
enh(style): update style of checkbox, radio, tabs

### PR DESCRIPTION
**Pull request by kduret**
## Description

Update style to follow material design : 
* checkboxes
* radio buttons
* tabs

**Fixes** MON-3813

## Type of change

- [ ] Patch fixing an issue (non-breaking change)
- [x] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software
- [ ] Updating documentation (missing information, typo...)

## Target serie

- [ ] 2.8.x
- [ ] 18.10.x
- [ ] 19.04.x
- [x] 19.10.x (master)

### CODE REVIEW

File: www/include/Administration/parameters/ldap/list.ihtml [hunk 1]
```diff
@@ -29,7 +29,12 @@
 	</table>
 	<table class="ListTable">
 		<tr class="ListHeader">
-			<td class="ListColHeaderPicker"><input type="checkbox" name="checkall" onclick="checkUncheckAll(this);"/></td>
+            <td class="ListColHeaderPicker">
```
Comment (loiclau) [inline www/include/Administration/parameters/ldap/list.ihtml]: indentation

File: www/include/Administration/performance/manageData.ihtml [hunk 2]
```diff
@@ -16,7 +16,12 @@
 	</table>
 	<table class="ListTable">
 	 	<tr class="ListHeader">
-	 		<td class="ListColHeaderPicker"><input type="checkbox" name="checkall" onclick="checkUncheckAll(this);"/></td>
+            <td class="ListColHeaderPicker">
```
Comment (loiclau) [inline www/include/Administration/performance/manageData.ihtml]: indentation

File: www/include/Administration/performance/viewMetrics.ihtml [hunk 3]
```diff
@@ -16,7 +16,12 @@
 	</table>
 	<table class="ListTable">
 	 	<tr class="ListHeader">
-	 		<td class="ListColHeaderPicker"><input type="checkbox" name="checkall" onclick="checkUncheckAll(this);"/></td>
+            <td class="ListColHeaderPicker">
```
Comment (loiclau) [inline www/include/Administration/performance/viewMetrics.ihtml]: indentation

File: www/include/configuration/configNagios/listNagios.ihtml [hunk 4]
```diff
@@ -34,7 +34,12 @@
 	</table>
 	<table class="ListTable">
 		<tr class="ListHeader">
-			<td class="ListColHeaderPicker"><input type="checkbox" name="checkall" onclick="checkUncheckAll(this);"/></td>
+            <td class="ListColHeaderPicker">
```
Comment (loiclau) [inline www/include/configuration/configNagios/listNagios.ihtml]: indentation

File: www/include/configuration/configObject/command/listCommand.ihtml [hunk 5]
```diff
@@ -31,9 +31,12 @@
 	</table>
 	<table class="ListTable">
 		<tr class="ListHeader">
-			<td class="ListColHeaderPicker">
-				<input type="checkbox" name="checkall" onclick="checkUncheckAll(this);"/>
-			</td>
+            <td class="ListColHeaderPicker">
```
Comment (loiclau) [inline www/include/configuration/configObject/command/listCommand.ihtml]: indentation

File: www/include/configuration/configObject/escalation/listEscalation.ihtml [hunk 6]
```diff
@@ -33,7 +33,12 @@
 </table>
 <table class="ListTable">
 	<tr class="ListHeader">
-		<td class="ListColHeaderPicker"><input type="checkbox" name="checkall" onclick="checkUncheckAll(this);"/></td>
+        <td class="ListColHeaderPicker">
```
Comment (loiclau) [inline www/include/configuration/configObject/escalation/listEscalation.ihtml]: indentation

File: www/include/configuration/configObject/host_dependency/listHostDependency.ihtml [hunk 7]
```diff
@@ -35,7 +35,12 @@
 
 	<table class="ListTable">
 		<tr class="ListHeader">
-			<td class="ListColHeaderPicker"><input type="checkbox" name="checkall" onclick="checkUncheckAll(this);"/></td>
+            <td class="ListColHeaderPicker">
```
Comment (loiclau) [inline www/include/configuration/configObject/host_dependency/listHostDependency.ihtml]: indentation

File: www/include/configuration/configObject/host_template_model/listHostTemplateModel.ihtml [hunk 8]
```diff
@@ -35,7 +35,12 @@
 	</table>
 	<table class="ListTable">
 		<tr class="ListHeader">
-			<td class="ListColHeaderPicker"><input type="checkbox" name="checkall" onclick="checkUncheckAll(this);"/></td>
+            <td class="ListColHeaderPicker">
```
Comment (loiclau) [inline www/include/configuration/configObject/host_template_model/listHostTemplateModel.ihtml]: indentation

File: www/include/configuration/configObject/hostgroup_dependency/listHostGroupDependency.ihtml [hunk 9]
```diff
@@ -33,7 +33,12 @@
 
 	<table class="ListTable">
 		<tr class="ListHeader">
-			<td class="ListColHeaderPicker"><input type="checkbox" name="checkall" onclick="checkUncheckAll(this);"/></td>
+            <td class="ListColHeaderPicker">
```
Comment (loiclau) [inline www/include/configuration/configObject/hostgroup_dependency/listHostGroupDependency.ihtml]: indentation

File: www/include/configuration/configObject/metaservice_dependency/listMetaServiceDependency.ihtml [hunk 10]
```diff
@@ -35,7 +35,12 @@
 
 	<table class="ListTable">
 		<tr class="ListHeader">
-			<td class="ListColHeaderPicker"><input type="checkbox" name="checkall" onclick="checkUncheckAll(this);"/></td>
+            <td class="ListColHeaderPicker">
```
Comment (loiclau) [inline www/include/configuration/configObject/metaservice_dependency/listMetaServiceDependency.ihtml]: indentation

File: www/include/configuration/configObject/service/listService.ihtml [hunk 11]
```diff
@@ -51,9 +51,12 @@
 	</table>
 	<table class="ListTable">
 		<tr class="ListHeader">
-			<td class="ListColHeaderPicker">
-				<input type="checkbox" name="checkall" onclick="checkUncheckAll(this);"/>
-			</td>
+            <td class="ListColHeaderPicker">
```
Comment (loiclau) [inline www/include/configuration/configObject/service/listService.ihtml]: indentation

File: www/include/configuration/configObject/servicegroup/listServiceGroup.ihtml [hunk 12]
```diff
@@ -33,7 +33,12 @@
 	</table>
 	<table class="ListTable">
 		<tr class="ListHeader">
-			<td class="ListColHeaderPicker"><input type="checkbox" name="checkall" onclick="checkUncheckAll(this);"/></td>
+            <td class="ListColHeaderPicker">
```
Comment (loiclau) [inline www/include/configuration/configObject/servicegroup/listServiceGroup.ihtml]: indentation

File: www/include/configuration/configObject/traps-groups/listGroups.ihtml [hunk 13]
```diff
@@ -34,7 +34,12 @@
 <input name="p" value="{$p}" type="hidden">
 	<table class="ListTable">
 		<tr class="ListHeader">
-			<td class="ListColHeaderPicker"><input type="checkbox" name="checkall" onclick="checkUncheckAll(this);"/></td>
+            <td class="ListColHeaderPicker">
```
Comment (loiclau) [inline www/include/configuration/configObject/traps-groups/listGroups.ihtml]: indentation

File: www/include/configuration/configResources/listResources.ihtml [hunk 14]
```diff
@@ -33,7 +33,12 @@
 	</table>
 	<table class="ListTable">
 		<tr class="ListHeader">
-			<td class="ListColHeaderPicker"><input type="checkbox" name="checkall" onclick="checkUncheckAll(this);"/></td>
+            <td class="ListColHeaderPicker">
```
Comment (loiclau) [inline www/include/configuration/configResources/listResources.ihtml]: indentation

File: www/include/monitoring/recurrentDowntime/listDowntime.html [hunk 15]
```diff
@@ -34,7 +34,12 @@
 
 <table class="ListTable">
 	<tr class="ListHeader">
-		<td class="ListColHeaderPicker"><input type="checkbox" name="checkall" onclick="checkUncheckAll(this);"/></td>
+        <td class="ListColHeaderPicker">
```
Comment (loiclau) [inline www/include/monitoring/recurrentDowntime/listDowntime.html]: indentation

File: www/include/options/accessLists/actionsACL/listsActionsAccess.ihtml [hunk 16]
```diff
@@ -25,7 +25,12 @@
 	</table>
 	<table class="ListTable">
 		<tr class="ListHeader">
-			<td class="ListColHeaderPicker"><input type="checkbox" name="checkall" onclick="checkUncheckAll(this);"/></td>
+            <td class="ListColHeaderPicker">
```
Comment (loiclau) [inline www/include/options/accessLists/actionsACL/listsActionsAccess.ihtml]: indentation

File: www/include/options/accessLists/menusACL/listsMenusAccess.ihtml [hunk 17]
```diff
@@ -27,7 +27,12 @@
 
 	<table class="ListTable">
 		<tr class="ListHeader">
-			<td class="ListColHeaderPicker"><input type="checkbox" name="checkall" onclick="checkUncheckAll(this);"/></td>
+            <td class="ListColHeaderPicker">
```
Comment (loiclau) [inline www/include/options/accessLists/menusACL/listsMenusAccess.ihtml]: indentation

File: www/include/options/accessLists/reloadACL/reloadACL.ihtml [hunk 18]
```diff
@@ -9,7 +9,12 @@
 	</table>
 	<table class="ListTable">
 		<tr class="ListHeader">
-			<td class="ListColHeaderPicker"><input type="checkbox" name="checkall" onclick="checkUncheckAll(this);"/></td>
+            <td class="ListColHeaderPicker">
```
Comment (loiclau) [inline www/include/options/accessLists/reloadACL/reloadACL.ihtml]: indentation

File: www/include/options/accessLists/resourcesACL/listsResourcesAccess.ihtml [hunk 19]
```diff
@@ -25,7 +25,12 @@
 	</table>
 	<table class="ListTable">
 		<tr class="ListHeader">
-			<td class="ListColHeaderPicker"><input type="checkbox" name="checkall" onclick="checkUncheckAll(this);"/></td>
+            <td class="ListColHeaderPicker">
```
Comment (loiclau) [inline www/include/options/accessLists/resourcesACL/listsResourcesAccess.ihtml]: indentation

File: www/include/options/media/images/listImg.ihtml [hunk 20]
```diff
@@ -24,13 +24,18 @@
...
 		</tr>
 	</table>
-	<table class="ListTable">
-		<tr class="ListHeader">
-			<td class="ListColHeaderPicker"><input type="checkbox" name="checkall" onclick="checkUncheckAll(this);"/></td>
-			<td class="ListColHeaderLeft">{$headerMenu_name}</td>
-			<td class="ListColHeaderLeft">{$headerMenu_img}</td>
-			<td class="ListColHeaderLeft">{$headerMenu_comment}</td>
-		</tr>
+    <table class="ListTable">
```
Comment (loiclau) [inline www/include/options/media/images/listImg.ihtml]: indentation

**Comment by kduret** [comment]
@loiclau indentation is ok
it is the whole file which has not proper indentation (tabs insteas of spaces)

**Pull request merged**